portfolio/src/main/webapp/js/smoothscroll.js

```diff
@@ -0,0 +1,40 @@
+$('document').ready(()=>{
```

Comment:
There are significant negative performance consequences to how you have implemented scrolling. You should consider the pros and cons carefully. In particular:

* using script to mediate scroll means that it won't work until the js is loaded and parsed.
* What happens if the user has JavaScript turned off?
* What happens if the user has a slow Internet connection and the script isn't loaded yet?

Reply:
The use of the anchor tag with the href set to the position on the site should provide an alternative to this, right? I tested this with javascript turned off and the scrolling is then just ignored and you are navigated to the section on the page like normal. What would your recommended approach be?

Reply:
I found a way using css that enables a smooth scroll that now renders this moot haha! Thank you!